javascripts-dev/src/ui.js

```diff
@@ -1,27 +1,64 @@
 var UserInterface = {
-  selector: "a[href=download]"
+  selector: 'a[href|="#download"]'
 };
 
 UserInterface.setup = function(zipper, downloader) {
 
-  var downloadLink = document.body.querySelector(UserInterface.selector);
+  var downloadLinks = document.body.querySelectorAll(UserInterface.selector);
 
-  var createZip = function() {
-    zipper.createZip(downloader);
+  var createZip = function(files) {
+    zipper.createZip(downloader, files);
   };
 
   var registerListener = function(link) {
+    console.log("adding listener to", link);
```

Comment:
I personally wouldn't include debugging code in commits that may end up in production, but that's maybe a personal choice.